<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyBoys</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 0, 0, 102), stop:0.326316 rgba(255, 0, 0, 130), stop:0.339799 rgba(255, 255, 255, 255), stop:0.636842 rgba(255, 255, 255, 135), stop:0.662469 rgba(0, 0, 255, 255), stop:1 rgba(0, 0, 255, 95));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="true_button">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>450</y>
       <width>120</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(79, 158, 0);</string>
     </property>
     <property name="text">
      <string>True</string>
     </property>
    </widget>
    <widget class="QPushButton" name="false_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>450</y>
       <width>120</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>False</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>111</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255,10);</string>
     </property>
     <property name="text">
      <string>Progress:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="known_word_lcd">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>30</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="time_lcd_word">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="word">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>720</width>
       <height>170</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>52</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>530</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255,10);</string>
     </property>
     <property name="text">
      <string>End Game</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
